modules/cluster-logging-configuration-of-json-log-data-for-default-elasticsearch.adoc

```diff
@@ -79,32 +79,38 @@ Suppose that you use the following snippet in your `ClusterLogForwarder` CR YAML
 
 [source,yaml]
 ----
-outputDefaults:
- elasticsearch:
-    structuredTypeKey: openshift.labels.myLabel <1>
+outputDefaults: <1>
+  elasticsearch:
+    structuredTypeKey: kubernetes.labels.managed <2>
     structuredTypeName: nologformat
+outputs:
+  - name: elasticsearch-secure
+    elasticsearch:
+      structuredTypeKey: openshift.labels.unmanaged <3>
+      structuredTypeName: nologformat
 pipelines:
- - name: application-logs
-   inputRefs:
-   - application
-   - audit
-   outputRefs:
-   - elasticsearch-secure
-   - default
-   parse: json
-   labels:
-     myLabel: myValue <2>
+  - name: application-logs
+    inputRefs:
+    - application
+    - audit
+    outputRefs:
+    - elasticsearch-secure
+    - default
+    parse: json
+    labels:
+      unmanaged: myvalue
 ----
-<1> Uses the value of the key-value pair that is formed by the OpenShift `myLabel` label.
-<2> The `myLabel` element gives its string value, `myValue`, to the structured log record.
+<1> `outputDefaults` applies only to the default Elasticsearch logstore and not to the other defined Elasticsearch outputs.
+<2> Use the value found at this path in the record to format the index name for the default output.
+<3> Use the value defined in pipeline to format the index name for the elasticsearch output named `elasticsearch-secure`.
 
-In that case, the following structured log record goes to the `app-myValue-write` index:
+In that case, the following structured log record goes to the `app-myvalue-write` index:
```

Comment:
"following structured log record goes to the ..." reads strange. Can you rephrase this? 

Something like this maybe (?):
"see the following log record for the `app-myvalue-write`..."

The placement of "In that case" (right after the call out description) reads out of context to me. Does it mean ----
"Suppose that you use the following snippet in your `ClusterLogForwarder` CR YAML file. In that case......."

Please consider adding a context or removing/revising "In that case....."

Reply:
Maybe we could rephrase as following:

"If you use the following snippet in your `ClusterLogForwarder` CR YAML file, see the following log record for the `app-myvalue-write` index:"

WDYT?

Reply:
Sounds good

Reply:
the log record doesn't have 'app-myvalue-write' field or even an index so "seeing the following log record is not correct. It should be something like:

The above CR will spec the following log record to be stored in the 'app-myvalue-write' index